Index: vue_music/.gitignore
===================================================================
diff --git a/vue_music/.gitignore b/vue_music/.gitignore
deleted file mode 100644
--- a/vue_music/.gitignore	(revision aa12a8372306a126706c7d2ddfeada77a838d737)
+++ /dev/null	(revision aa12a8372306a126706c7d2ddfeada77a838d737)
@@ -1,23 +0,0 @@
-.DS_Store
-node_modules
-/dist
-
-
-# local env files
-.env.local
-.env.*.local
-
-# Log files
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
-pnpm-debug.log*
-
-# Editor directories and files
-.idea
-.vscode
-*.suo
-*.ntvs*
-*.njsproj
-*.sln
-*.sw?
