Index: NeteaseCloudMusicApi-4.28.0/.idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/NeteaseCloudMusicApi-4.28.0/.idea/.gitignore b/NeteaseCloudMusicApi-4.28.0/.idea/.gitignore
deleted file mode 100644
--- a/NeteaseCloudMusicApi-4.28.0/.idea/.gitignore	(revision e9ca5c760e5d9bf86ee5024e447b238415bbfdd5)
+++ /dev/null	(revision e9ca5c760e5d9bf86ee5024e447b238415bbfdd5)
@@ -1,10 +0,0 @@
-# 默认忽略的文件
-/shelf/
-/workspace.xml
-# 基于编辑器的 HTTP 客户端请求
-/httpRequests/
-# 依赖于环境的 Maven 主目录路径
-/mavenHomeManager.xml
-# Datasource local storage ignored files
-/dataSources/
-/dataSources.local.xml
